src/assets/imageLinks.js

```diff
@@ -0,0 +1,32 @@
+const images = {
+  frodo: 'https://www.looper.com/img/gallery/frodo-baggins-entire-backstory-explained/intro-1582640416.jpg',
```

Comment:
I know it's tedious, but it'll be really nice to have the photos. I'm hoping I'll have time to do something similar for my drivers.